lambdas/accesstoken/src/main/java/uk/gov/di/ipv/cri/passport/accesstoken/AccessTokenHandler.java

```diff
@@ -63,7 +63,7 @@ public APIGatewayProxyResponseEvent handleRequest(
             APIGatewayProxyRequestEvent input, Context context) {
         LogHelper.attachComponentIdToLogs();
         try {
-            tokenRequestValidator.authenticateClient(input.getBody());
+            String jwtId = tokenRequestValidator.authenticateClient(input.getBody());
```

Comment:
Could this var be called something else, like clientAuthJwtId, just to make it a little clearer where this ID comes from